docs/source/databases.rst

```diff
@@ -12,7 +12,7 @@ NCBI
 
 * taxdump.tar.gz - `link <ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz>`__
     
-    - required files within taxdump tarchive are *nodes.dmp*, *names.dmp* and *merged.dmp*
+    - required files within taxdump archive are *nodes.dmp*, *names.dmp* and *merged.dmp*
```

Comment:
`taxdump.tar.gz` is a gzipped tar archive. Perhaps tarball would be a better name? I think this is more often referred to as a tarball rather than tarchive

Reply:
Hijacking the comment just to drop a fun fact:  

tar stands for **t**ape **ar**chive and like @WiscEvan said, they are commonly called *tarballs*